data/pigui/libs/equipment-outfitter.lua

```diff
@@ -0,0 +1,648 @@
+	local rounding = styles.ItemCardRounding
+
+	local iconOffset = framePadding + Vector2(rounding, 0)
+	local textOffset = iconOffset   + Vector2(icon_size.x + framePadding.x, pionillium.heading.size * 0.25)
+	local fontCol = ui.theme.colors.font
+
+	local startPos = ui.getCursorScreenPos()
+
+	variant = variant or ui.theme.buttonColors.dark
+	local clicked = false
+
+	ui.withButtonColors(variant, function()
+		pigui.PushStyleVar("FramePadding", framePadding)
+		pigui.PushStyleVar("FrameRounding", rounding)
+		clicked = pigui.Button("##" .. label, size)
+		pigui.PopStyleVar(2)
+	end)
+
+	if ui.isItemHovered() then
+		local tl, br = ui.getItemRect()
+		ui.addRectFilled(tl, tl + Vector2(rounding, size.y), fontCol, 4, 0x5)
+	end
+
+	ui.withFont(pionillium.heading, function()
+		ui.addIconSimple(startPos + iconOffset, icon, icon_size, fontCol)
+		ui.addText(startPos + textOffset, fontCol, label)
+
+		if infoText then
+			local calcSize = ui.calcTextSize(infoText)
+			local infoTextOffset = Vector2(size.x - (framePadding.x + calcSize.x), textOffset.y)
+			ui.addText(startPos + infoTextOffset, fontCol, infoText)
+		else
+			local endOffset = size.x - framePadding.x
+			local width = ui.getTextLineHeight() / 1.6
+			ui.addRectFilled(
+				startPos + Vector2(endOffset - width, framePadding.y),
+				startPos + Vector2(endOffset, height - framePadding.y),
+				variant.normal, 4, 0)
+		end
+	end)
+
+	return clicked
+end
+
+--=============================================================================
+
+local EquipCardAvailable = EquipCard.New()
+EquipCardAvailable.tooltipStats = false
+
+---@class UI.EquipmentOutfitter.EquipData : UI.EquipCard.Data
+---@field canInstall boolean
+---@field canReplace boolean
+---@field available boolean
+---@field price number
+
+---@class UI.EquipmentOutfitter.EquipCard : UI.EquipCard
+local EquipCardUnavailable = EquipCard.New()
+
+EquipCardUnavailable.tooltipStats = false
+EquipCardUnavailable.colors = ui.theme.buttonColors.deselected
+EquipCardUnavailable.textColor = ui.theme.styleColors.danger_300
+
+---@param data UI.EquipmentOutfitter.EquipData
+function EquipCardUnavailable:tooltipContents(data, isSelected)
+	EquipCard.tooltipContents(self, data, isSelected)
+
+	ui.spacing()
+
+	ui.withStyleColors({ Text = self.textColor }, function()
+		ui.withFont(pionillium.details, function()
+
+			if not data.canInstall then
+				ui.textWrapped(l.NOT_SUPPORTED_ON_THIS_SHIP % { equipment = data.name } .. ".")
+			elseif not data.canReplace then
+				ui.textWrapped(l.CANNOT_SELL_NONEMPTY_EQUIP .. ".")
+			else
+				ui.textWrapped(l.YOU_NOT_ENOUGH_MONEY)
+			end
+
+		end)
+	end)
+end
+
+--=============================================================================
+
+---@class UI.EquipmentOutfitter : UI.Module
+---@field New fun(): self
+local Outfitter = utils.class("UI.EquipmentOutfitter", Module)
+
+Outfitter.SortKeys = {
+	{ id = "name", label = le.SORT_NAME },
+	{ id = "mass", label = le.SORT_WEIGHT },
```

Comment:
Should use Mass, not weight..

Reply:
I'll change this one before merge  in the interest of not introducing a newly-unused translation string.

Reply:
Removed the `SORT_XXX` translation strings in favor of using existing keys from `ui-core`.